db/migrations/20220120000700_add_constraints.up.go

```diff
@@ -48,8 +49,18 @@ func init() {
 					--   This can happen when two transactions try to access the same account
 					FOR UPDATE NOWAIT;
 
-					-- If it is an incoming account return; otherwise check the balance
-					IF debit_account_type IS NULL
+					-- check if credit_account type is fees, if it's fees we don't check for negative balance constraint
+					SELECT INTO credit_account_type type
```

Comment:
just did the same as above, checked if credit account type is fees, if it is don't check for sum constraint, is this enough?